### PR TITLE
Added ability to change button font color

### PR DESCRIPTION
**Pull request by LodeKennes**
# Overview

I added a way to change the font color of the buttons, up to now you were only able to change the other colors in the calendar itself.

# Test Plan

I implemented the component in my own code and tried it out.

image

### CODE REVIEW

**Comment by LodeKennes** [comment]
Hi @mmazzarolo, any idea when this could be merged in? Thanks in advance.

File: typings/index.d.ts [hunk 1]
```diff
@@ -31,6 +31,13 @@ export type HeaderComponent = React.ComponentType<{
 export type PickerComponent = React.ComponentType<IOSNativeProps>;
 
 export interface DateTimePickerProps {
+  /**
+   * The color of text on the buttons on iOS
```
Comment (mmazzarolo) [inline typings/index.d.ts:35]:
```suggestion
    * iOS buttons text color
````

File: README.md [hunk 2]
```diff
@@ -92,7 +92,8 @@ export default Example;
 👉 Please notice that **all the [`@react-native-community/react-native-datetimepicker`](https://github.com/react-native-community/react-native-datetimepicker) props are supported** as well!
 
 | Name                      | Type      | Default      | Description                                                                                         |
-| ------------------------- | --------- | ------------ | --------------------------------------------------------------------------------------------------- |
+|---------------------------|-----------| ------------ |-----------------------------------------------------------------------------------------------------|
+| `buttonFontColor`         | string    |              | The color of the confirm button texts (iOS)                                                         |
```
Comment (mmazzarolo) [inline README.md:96]: Can we rename it to `buttonTextColorIOS`?

**Review by mmazzarolo — CHANGES_REQUESTED** [review]
Sorry, I've been pretty busy lately 😓 A couple of suggestion but generally looks good to me. Thanks!

File: README.md (hunk 2 above)
Comment (LodeKennes) [reply README.md]: @mmazzarolo Thanks for the feedback! Done! 👍

**Review by mmazzarolo — APPROVED** [review]

**Comment by mmazzarolo** [comment]
Thank you @LodeKennes ! <3

**Pull request merged**